vars/executeDockerOnKubernetes.groovy

```diff
@@ -0,0 +1,90 @@
+import com.cloudbees.groovy.cps.NonCPS
```

Comment:
naming needs to be aligned, e.g. `dockerExecuteOnKubernetes`. The current name is unlike the agreed pattern